Docs update: Add nominated account endpoints to savings account page (#400)
clearbank/clearbank.github.io: content/uk/docs/embedded-banking/savings-accounts.mdx

## Savings accounts

### Overview

> A savings account is an FSCS-protected bank account that can accrue interest on behalf of a retail or business customer. Savings accounts may be held individually or jointly, and you can update their interest rate configuration at will.

Reviewer: ```suggestion
A savings account is an FSCS-protected bank account that can accrue interest on behalf of a retail or business customer. Savings accounts may be held individually or jointly.
```

> ClearBank calculates interest accruals on savings account deposits based on pre-agreed interest configurations. These accruals are posted for you as regular interest payments to your customers' savings accounts. To learn more about interest configurations, refer to [Interest configuration](./interest-configuration).

Reviewer: ```suggestion
ClearBank calculates interest accruals on savings account deposits based on pre-agreed interest configurations. You can update the interest rate on your savings accounts to a pre-agreed configuration at any time.

Interest accruals are posted for you as regular payments to your customers' savings accounts. To learn more about interest configurations, refer to [Interest configuration](./interest-configuration).
```

</Callout>

#### Message flow diagram

![Submit the POST /v2/savings endpoint. Once you receive a 201 response, submit the POST /v1/accounts/{accountId}/related-accounts to set up a nominated account. Once you receive a 201 response, funds can now be added to the savings account.](/assets/images/savings-account-open.svg)

### Create joint savings accounts

Both single and joint savings accounts can be created using the [POST /v2/savings](#create-a-savings-account-endpoint) endpoint. To make joint accounts using this endpoint, specify each customer's ID in the `customers` array.

#### Example payload: two customers

```json
{
  "ownerName": "John and Jane Smith",
  "sortCode": "010203",
  "productId": "1a2b3c4d-5e6f-7a8b-9c0d-1e2f3a4b5c6d",
  "customers": [
    {
      "customerId": "0f1e2d3c-4b5a-6c7d-8e9f-0a1b2c3d4e5f"
    },
    {
      "customerId": "abcdef12-3456-7890-1234-56789abcdef0"
    }
  ],
  "interestConfigurationId": "1234abcd-5678-9ef0-1234-56789abcdef0"
}
```

#### Example payload: four customers

> While joint accounts will typically have two customers, we can support any number of customers where there is a valid use case.

Reviewer: Do they have to discuss this with us before we can permit it on their account or is this out-the-box functionality?

Author: Good Q. Sec

Author: now addressed

  ]}
/>

<WebhookPlaceholder render='account-created-webhook-v1' />

## Manage nominated accounts

ClearBank maintains two methods of managing nominated accounts. The first, newer method uses four `related-accounts` endpoints. These endpoints are also used to manage Hub Accounts.

To update a nominated account, you can call the [DELETE /v1/accounts/{accountId}/related-accounts/{iban}](#remove-a-nominated-account) followed by the [POST /v1/accounts/{accountId}/related-accounts](#add-a-nominated-account).

> The second - older - method uses just `nominated-account` endpoints:

Reviewer: ```suggestion
The second legacy method uses just `nominated-account` endpoints:
```
